awx/api/urls/job.py

```diff
@@ -27,6 +28,7 @@
     re_path(r'^(?P<pk>[0-9]+)/create_schedule/$', JobCreateSchedule.as_view(), name='job_create_schedule'),
     re_path(r'^(?P<pk>[0-9]+)/job_host_summaries/$', JobJobHostSummariesList.as_view(), name='job_job_host_summaries_list'),
     re_path(r'^(?P<pk>[0-9]+)/job_events/$', JobJobEventsList.as_view(), name='job_job_events_list'),
+    re_path(r'^(?P<pk>[0-9]+)/job_events/children_summary/$', JobJobEventsChildrenSummary.as_view(), name='job_job_event_children_summary'),
```

Comment:
Does the UI really need this specific to jobs, or would it be easier to apply it to all unified jobs, including project updates and ad-hoc commands? Your logic (like `job.get_event_queryset()`) could mostly work for all job types. Best to ask these questions before making the schema change.

Reply:
@keithjgrant is this only specific to Jobs? Would it be best to have children_summary endpoint for other types of jobs?

ProjectUpdateEvent does not have a parent_uuid field

Reply:
Currently the UI only shows the collapsible view for playbook jobs. If we want that functionality for any other job types, I wouldn't be opposed, but they would also need their parent_uuid's populated, which we don't have now